oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -684,6 +735,18 @@ public Builder setQuotaProjectId(String quotaProjectId) {
       return this;
     }
 
+    @CanIgnoreReturnValue
+    public Builder setTransport(AuthTransport transport) {
+      this.transport = transport;
+      return this;
+    }
+
+    @CanIgnoreReturnValue
+    public Builder setBindingEnforcement(BindingEnforcement bindingEnforcement) {
+      this.bindingEnforcement = bindingEnforcement;
+      return this;
+    }
```

Comment:
can you some small/ simple javadocs for these public setters?

Reply:
Done in https://github.com/googleapis/google-auth-library-java/pull/1558/commits/8a2d967552c6322f7b6514dcde4b7ba190a7128a